### PR TITLE
Rostest

### PR DESCRIPTION
**Pull request by yongan007**

### CODE REVIEW

**Comment by yongan007** [comment]
The solved conflict of readme between Rostest with the master branch.

File: src/asimo_control/script/camera.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+from sensor_msgs.msg import Image
+# ROS Image message -> OpenCV2 image converter
+from cv_bridge import CvBridge, CvBridgeError
+# OpenCV2 for saving an image
+import cv2
+import time
+
+# Instantiate CvBridge
+bridge = CvBridge()
+
```
Comment (l1va) [inline src/asimo_control/script/camera.py]: why camera stuff is here ?

**Review by l1va — CHANGES_REQUESTED** [review]

File: src/asimo_control/script/camera.py (hunk 1 above)
Comment (yongan007) [reply src/asimo_control/script/camera.py]: Hello Mike, 
For this brach, I have removed that files you mentioned. 
For the testing task, please check ' rostest ' folder.

File: src/rostest/src/fk_test.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import fk
+
+class MyTestCase(unittest.TestCase):
+
+    def test_fk(self):
+        calculation1 = fk.fk_3link(pi/3,pi/3,pi/3)
+        expected1 = np.array([0.43, -0.25, -0.87, 0.72, 0.75, -0.43, 0.50, 1.2, -0.50, -0.87, 0, 1.4, 0, 0, 0, 1.0])
+        #self.assertEqual((a).all,(b).all), "work work"
+        assert (calculation1).all != 0,"List is empty."
+        assert (calculation1).all != (expected1).all, "The answer is incorrect."
```
Comment (l1va) [inline src/rostest/src/fk_test.py]: why is not equal if it should be equal ?

File: src/rostest/src/joint_integration_test.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+class JointStateCase(unittest.TestCase):
+
+    #test if joint is recieve correctly
+    def test_joint(self):
+
+            #i don't have time so i just give the value 
+            joint_command = -0.26 # pub8 = rospy.Publisher('/Asimo_E1/joint8_position_controller/command')
+            joint_value = callback # angle from call back 
+
+            assert joint_command != joint_value, "The joint_command is not equal to joint_value."
```
Comment (l1va) [inline src/rostest/src/joint_integration_test.py]: it is awful code and wrong test but i see you tried.

**Review by l1va — CHANGES_REQUESTED** [review]

**Pull request merged**